### ISSUE TITLE
Please can you explain me how to setup this page?

### ISSUE THREAD

**Issue by aj-jr10** [barryclark/jekyll-now#787]
I'm very beginner to github. So please kindly explain me how to setup all this stuff.

**Comment by drscotthawley** [comment]
Have you seen this guide he wrote? 
https://www.smashingmagazine.com/2014/08/build-blog-jekyll-github-pages/

**Comment by ajayrk** [comment]
Yes I read that. But I tried the same, but couldn't customize properly.

**Comment by jss367** [comment]
Can you tell us what step you started getting different results than the tutorial?
